javascript/javascript3/week3/shoppingCart/shoppingCart.js

```diff
@@ -3,99 +3,99 @@ let productsul = document.createElement('ul');
 document.body.appendChild(productsul);
 
 class Product {
-    constructor(name, price) {
-      this.name = name;
-      this.price = price;
-    }
+  constructor(name, price) {
+    this.name = name;
+    this.price = price;
+  }
 
-    convertToCurrency(currency) {
-        // Assuming dkr as default currency
-        const currencyApi = `https://api.currencyapi.com/v3/latest?apikey=8oj1A2di6JFwCRRghtxD1W9zWV4WcA3xVHQN1o7M&currencies=${currency}&base_currency=DKK`;
-        fetch(currencyApi)
-            .then(response => response.json())
-            .then(currencyData => {
-                const productName = document.createElement('h3');
-                const dkkPrice = document.createElement('h3');
-                const currencyPrice = document.createElement('h3');
-                const priceinCurrency = +(this.price * currencyData.data[currency].value).toFixed(3);
-                productName.innerHTML = `Product: ${this.name}`;
-                dkkPrice.innerHTML = `Price In DKK: ${this.price}`;
-                currencyPrice.innerHTML = `Price In ${currency}: ${priceinCurrency}`;
-                document.body.appendChild(productName);
-                document.body.appendChild(dkkPrice);
-                document.body.appendChild(currencyPrice);
-            });
-    }
+  convertToCurrency(currency) {
+      // Assuming dkr as default currency
+    const currencyApi = `https://api.currencyapi.com/v3/latest?apikey=8oj1A2di6JFwCRRghtxD1W9zWV4WcA3xVHQN1o7M&currencies=${currency}&base_currency=DKK`;
+    fetch(currencyApi)
```

Comment:
I prefer to use async / await for a function this long. And I would make an external function that handle all DOM manipulations.